src/components/inputs/DatePickerComponent.tsx

```diff
@@ -18,40 +19,78 @@ export const DatePickerComponent = ({ getDate, dateValue }: Prop) => {
     return date ? date.format('MMM DD, YYYY') : '' // Format the date as "Mar 08, 2024"
   }
 
+  const [anchorEl, setAnchorEl] = React.useState<null | HTMLElement>(null)
+
+  const handleClick = (event: React.MouseEvent<HTMLElement>) => {
+    setAnchorEl(anchorEl ? null : event.currentTarget)
+  }
+
+  const open = Boolean(anchorEl)
+  const id = open ? 'calender-element' : undefined
```

Comment:
can't we use `null` instead of `undefined` @aatbip

Reply:
I am using `undefined` because of [this](https://mui.com/material-ui/react-popper/) recommendation.

Reply:
@saugatmaharjan I am not able to use `null` here. Getting type error since `id` is used as attribute for `Popper` component which should have type `string | undefined`.

Reply:
Ok